LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs

```diff
@@ -3818,6 +3818,54 @@ private static void GetNuvotonConfigurationD(ISuperIO superIO, Manufacturer manu
 
                         break;
 
+                    case Model.ROG_STRIX_Z790_E_GAMING_WIFI: //NCT6798D
+
+                        // Volts
+                        v.Add(new Voltage("CPU Core Voltage", 0));
```

Comment:
You might get notified that this should get renamed to `Vcore` instead of this. Although what you put makes a bit more sense to name this way in my opinion.

We'll see what PhyxionNL thinks

Reply:
CPU Core would be OK, but then I prefer to see it renamed everywhere. I don't see a point in naming things double (a voltage is already a voltage :))